docs/faq/questions/cloud-faq.mdx

```diff
@@ -200,66 +249,125 @@ parallel. If there is NO common variable, try using the commit SHA string.
+Yes! The test is captured in Cypress Cloud as it executed in your CI run. That means the replay will show things like [slow typing](/api/cypress-api/keyboard-api#Slow-down-typing-by-increasing-the-keystroke-delay) or any [`cy.wait()`](/api/commands/wait) included in your test.
 
-**If you are using GitHub Enterprise or GitLab for Enterprise (Self-managed),
-you may also need to add both the following to the version control IP allow
-list:**
+### <Icon name="angle-right" /> Are there performance implications when using Test Replay?
 
-- `3.211.102.119` - **Dedicated IP**
-- `18.213.72.78` - **Dedicated IP**
+Expect an even performance exchange with Test Replay vs. video. There is more activity to capture and replay tests but without the overhead of recording, compressing, and storing video assets. `video` capture is set to false by default starting in Cypress `v13`. See the full [v13 changelog](/guides/references/changelog#12-0-0).
+
+### <Icon name="angle-right" /> Is every aspect of my stack included in Test Replay?
+
+Our aim is to create an impactful debugging experience that covers the most ground for users. However, if you have a need that would make Test Replay even more valuable, please [submit an issue](https://github.com/cypress-io/cypress/issues/new?assignees=&labels=&projects=&template=4-feature.yml).
+
+We will continue to expand and can see a need to support (to name a few):
+
+- Web sockets
+- Canvas elements
+- Shadow DOM
+- Server side events
+
+### <Icon name="angle-right" /> Can I use Test Replay for tests recorded in different browsers?
+
+Test Replay leverages [Chrome DevTools Protocol(CDP)](https://chromedevtools.github.io/devtools-protocol/), so currently supports Chromium-based browsers (Chrome, Edge, and Electron) only.
+
+Test Replay would be disabled, with a message that it's only available on Chromium, for tests run in Firefox or Webkit (Safari). You can still record and capture test [artifacts](/guides/cloud/debugging/recorded-runs#Artifacts) (screenshots, videos and CI logs) via other browsers in separate [run groups](/guides/cloud/smart-orchestration/parallelization#Grouping-test-runs).
+
+### <Icon name="angle-right" /> Can I replay tests from historical Cypress Cloud runs?
+
+Test Replay is available for tests recorded using Cypress `v13` and up. Tests recorded prior to this will not have Test Replay enabled in Cypress Cloud. You will still have any artifacts collected during the pre `v13` test runs.
+
+### <Icon name="angle-right" /> Can I share my test replays?
+
+Yes! Share among your team or across teams, like QA, marketing, design, etc. Just open the replay, copy the url from the browser address bar and pass it along.
+
+### <Icon name="angle-right" /> Will Test Replay write any assets to the file system?
+
+Yes, but it is stored in a temporary directory and deleted after the run. There is nothing to `.gitignore`.
+
+### <Icon name="angle-right" /> Can I `console.log()` data in the Developer Tools panel?
```

Comment:
Test Replay. It can clarify, the "console view of the dev tools panel".

This one may be a stretch. It came up during one of our first demos with engineering. Just adding a clarifying note about what to expect since the section is new and logging may be suboptimal under certain circumstances.